packages/menu/src/MenuList/MenuList.js

```diff
@@ -0,0 +1,106 @@
+import React, { useCallback, useEffect, useRef } from 'react'
+import PropTypes from 'prop-types'
+import styled, { css } from 'styled-components'
+import { Flex, applySizes } from 'pcln-design-system'
+
+const sizes = {
+  singleColumn: css`
+    & > * {
+      width: 100%;
+    }
+  `,
+  twoColumns: css`
+    & > * {
+      width: calc(50% - 4px);
+    }
+  `,
+}
+
+const MenuContainer = styled(Flex)`
+  font-family: 'Montserrat', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+  overflow-y: scroll;
+  height: ${(props) => props.height || 300}px;
+
+  & > * {
+    margin-right: 4px;
+    margin-bottom: 4px;
+  }
+
+  ${applySizes(sizes)};
+`
+
+function focusNext(item) {
+  return item && item.nextElementSibling
+}
+
+function focusPrevious(item) {
+  return item && item.previousElementSibling
+}
+
+function moveFocus(currentFocus, traversalFunction) {
+  const nextFocus = traversalFunction(currentFocus)
+  nextFocus && nextFocus.focus()
+}
+
+function MenuList({ children, color, size, height, handleClose }) {
+  const listRef = useRef(null)
+  const currentItemRef = useRef(null)
+
+  useEffect(() => {
+    currentItemRef.current?.focus()
+  }, [])
+
+  const handleKeyDown = useCallback((evt) => {
+    const list = listRef.current
+    const key = evt.key
+
+    const currentFocus = list?.ownerDocument?.activeElement
+
+    if (key === 'ArrowDown') {
+      evt.preventDefault()
+      moveFocus(currentFocus, focusNext)
+    } else if (key === 'ArrowUp') {
+      evt.preventDefault()
+      moveFocus(currentFocus, focusPrevious)
+    }
+  }, [])
```

Comment:
So for this, focus actually needs to stay at the list level (The container with the `role="list"`). Each menu item needs to have an `aria-selected` and the state will be updated to false with the logic above. 
Keeping the focus at the list container level allows the screen reader to read which element is currently selected and the position in the list (ie. "USD selected, 1 of 4")